deploy/role.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can the `deploy` directory be deleted and a single YAML made available in the parent folder for launching both the operators?

Reply:
These are automatically generated files. These are not used anywhere now. To deploy ndm-daemonset and ndm-operator, we have to apply single yaml (ndm-operator.yaml).